platform.txt

```diff
@@ -26,7 +26,7 @@ compiler.debug.flags=
 
 compiler.c.flags=-c {compiler.debug.flags} {compiler.both.flags} {compiler.preproc.flags} -std=gnu11
 
-compiler.cpp.flags=-c {compiler.debug.flags} {compiler.both.flags} {compiler.preproc.flags} -std=gnu++17 -fpermissive 
+compiler.cpp.flags=-c {compiler.debug.flags} {compiler.both.flags} {compiler.preproc.flags} -std=gnu++17 -fpermissive "-I{runtime.platform.path}/libraries/SPI/src"
```

Comment:
There should be no need to add an include path here, and the Arduino builder will automatically find the library dependencies and include the path.